src/space.h

```diff
@@ -121,6 +121,31 @@ struct Space {
         case (ACTIVE):
             f = [molid](Tgroup &i) { return (i.id == molid) && (i.size() == i.capacity()); };
             break;
+        case (ALL_NEUTRAL):
```

Comment:
Compilers are smart nowadays. Nevertheless I would sum the charges only if previous part of condition is true. See also the other cases below.